_posts/2016-05-09-introducting-phpstorm-integration.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Introducing PHPStorm integration"
+date: 2016-05-09 10:00:00
+author: "@agallou"
+categories: release
+---
+
+Introduction
+------------
+
+Until recently, atoum were not integrated in Jetbrain's IDE, [PhpStorm](https://www.jetbrains.com/phpstorm/).
+
+Since several weeks, this is no longer the case : atoum's plugin for PHPStorm is live.
+
+We're gonna take a tour of the functionalities of this plugin.
+
+
+Go from test to the tested class
+--------------------------------
+
+When writing unit tests, en moreover when doiing TDD, we often switch from the test to the tested class. 
+
+The plugin adds an icon in from of the name of the tested class to go to test class. It also does the opposite and adds in icon in from of the test class.
+
+A shortcut is also available for a faster usage of this functionality.
+
+![Switch](/images/posts/2016-05-09-introducting-phpstorm-integration/switch-icon.png)
+
+
+Test identification
+-------------------
+
+The file icon is changed to add atoum's logo when the file contains a unit test.
+
+This will allow you to easily disinguish the test file from the tested file when those are displayed on the tabs.
+
+![Tabs](/images/posts/2016-05-09-introducting-phpstorm-integration/custom_icon-tabs.png)
+
+
+Running tests
+-------------
+
+Finally, you now can your directy your tests from the IDE. The results will be displayed as nice treeview, listing witch methods have passed or failed (and the that case, with the error details):
+
+<style type="text/css">
+    img[alt="Run"] {
+        max-width: 100%;
+    }
+</style>
+
+![Run](/images/posts/2016-05-09-introducting-phpstorm-integration/run.png)
+
+
+Conclusion
+----------
+
+As we seen, the plugin will allow to write your tests faster, without even leaving your IDE.
+
+You can install the plugin, directy from the IDE. Installation instructions could be be found in the project's [README](https://github.com/atoum/phpstorm-plugin).
+
+Don't hesitate to contribute, ether by dooing a pull request, adding an [issue](https://github.com/atoum/phpstorm-plugin/issues) or giving is feedback on your the usage of the plugin. 
```

Comment:
`s/ether/either/`
`s/dooing/submitting/`
`s/adding/opening/`
"or giving **us**"
"your **~~the~~** usage"